cloudformation/batch-substack.cfn.json

```diff
@@ -120,7 +120,7 @@
           ]
         },
         "ManagedPolicyArns": [
-          "arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess",
+          "arn:aws:iam::aws:policy/AmazonS3FullAccess",
```

Comment:
Isn't it too much?

Reply:
Also the `arn:aws:` won't work in non-commercial partitions, I know batch isn't supported there yet but we should make changes as if it is. Please use `${AWS::Partition}`